client/app/telecom/pack/migration/building-details/telecom-pack-migration-building-details.html

```diff
@@ -7,35 +7,89 @@
+                                data-title="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
+                                placeholder="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
+                                data-items="$ctrl.model.selectedStair.floors" data-align="start" data-match="label">
+                        <span data-ng-bind="$item.label"></span>
+                    </oui-select>
+                </div>
             </div>
+        </div>
 
-            <div data-ng-if="$ctrl.model.selectedStair !== null">
-                <span data-translate="telecom_pack_migration_building_details_select_floor"></span>
-                <oui-select name="floor" data-model="$ctrl.model.selectedFloor"
-                            data-title="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
-                            placeholder="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
-                            data-items="$ctrl.model.selectedStair.floors" data-align="start" data-match="label">
-                    <span data-ng-bind="$item.label"></span>
-                </oui-select>
+        <div>
+            <h3 data-translate="telecom_pack_migration_pto_title"></h3>
+            <div data-translate="telecom_pack_migration_pto_description"></div>
+            <div class="row">
+                <div class="col-xs-6 col-md-6">
+                    <div class="radio">
+                        <label for="radioPtoYes">
+                            <input type="radio"
+                                    id="radioPtoYes"
+                                    data-ng-model="$ctrl.model.pto"
+                                    data-ng-value="true"/>
+                            <span data-translate="telecom_pack_migration_pto_check_yes"></span>
+                        </label>
+                    </div>
+                </div>
+                <div class="col-xs-6 col-md-6">
+                    <div class="radio">
+                        <label for="radioPtoNo">
+                            <input type="radio"
+                                    id="radioPtoNo"
+                                    data-ng-model="$ctrl.model.pto"
+                                    data-ng-value="false"/>
+                            <span data-translate="telecom_pack_migration_pto_check_no"></span>
+                        </label>
+                    </div>
+                </div>
+            </div>
+            <div class="row" ng-if="$ctrl.model.pto">
+                <div class="col-xs-12 col-md-6">
+                    <div class="radio">
+                        <label for="referencePto">
+                            <span data-translate="telecom_pack_migration_pto_reference"></span>
+                            <input type="text" id="referencePto" data-ng-model="$ctrl.model.ptoReference"/>
```

Comment:
As you wrap your control with `<div class="radio">`, it must be an `<input type="radio>"`. 

Also, why do not prefer using: [`<oui-radio>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/radio) component?